mappings/net/minecraft/client/texture/Sprite.mapping

```diff
@@ -60,3 +68,17 @@ CLASS net/minecraft/class_1058 net/minecraft/client/texture/Sprite
 		FIELD field_21758 images [Lnet/minecraft/class_1011;
 		METHOD <init> (Lnet/minecraft/class_1058;Lnet/minecraft/class_1058$class_4727;I)V
 			ARG 3 mipmap
+		METHOD method_24128 apply ()V
+			COMMENT Linearly interpolate between the current and next frame on all miplevels
+			COMMENT based on the tick position within the current frame,
+			COMMENT and upload the results to the currently bound texture to the frame slot at position (0,0).
+		METHOD method_24129 mix (DII)I
+			ARG 1 alpha
```

Comment:
`alpha`? We use `delta` for this in `lerp`.

Reply:
As I said, this is based on glsl terminology (they use just `a` but it's clearly based on blending language, + this lerp is directly used  for blending).

Reply:
Think `alpha` for a blend of a background and a foreground color is fine